admin_manual/configuration/user/user_types.rst

```diff
@@ -0,0 +1,117 @@
+==============
+ownCloud Roles
+==============
+
+ownCloud supports six user types.
```

Comment:
below I count 8 :)

Reply:
And should call it "roles" there as well. Maybe something like: "ownCloud is designed to be used by separated roles depending on the size of the setup"